Gravatar/DefaultImageType.cs

```diff
@@ -11,12 +11,14 @@ public enum DefaultImageType
         /// </summary>
         None = 0,
 
+/*
```

Comment:
:bulb: `#if false`

Reply:
Curious: why do you prefer a directive?

Reply:
Directives support nesting and also you can change `false` to `true` to see what's inside.